zerver/tests/test_delete_unclaimed_attachments.py

```diff
@@ -55,6 +56,10 @@ def assert_exists(
             ArchivedAttachment.objects.filter(id=attachment.id).exists(), has_archived_attachment
         )
 
+    def get_and_do_delete_old_unclaimed_attachments(self, weeks_ago: int) -> None:
+        old_attachments, old_archived_attachments = get_old_unclaimed_attachments(weeks_ago)
+        do_delete_old_unclaimed_attachments(old_attachments, old_archived_attachments)
+
```

Comment:
This new function is only introduced to achieve consistency and to avoid having to write `get_old_unclaimed_attachments` each time we call  `do_delete_old_unclaimed_attachments`  in each of the test cases.